styleguide/source/assets/scss/05-pages/_people-listing.scss

```diff
@@ -6,16 +6,24 @@
   &__card-container {
     @include gutter($padding-top-full...);
     @include gutter($padding-bottom-full...);
-    display: flex;
-    flex-wrap: wrap;
+
```

Comment:
```suggestion
```

Reply:
@designerneil can you accept this code styling change?

Reply:
@JaeBea I don't understand why those need to get removed?

Reply:
The extra space?